_raml/apps/products/examples/get_product_productid_response.json

```diff
@@ -88,6 +88,16 @@
   ],
   "stocklevel": 100,
   "minStocklevel": 10,
+  "videos": [
+    {
+      "type": "YouTube",
+      "source": "08154713"
```

Comment:
shouldn't this be a (fake) URL to make the example more realistic? Like www.youtube.com/dsjögkh4df??